src/main/scala/mesosphere/marathon/api/v2/json/V2AppDefinition.scala

```diff
@@ -42,6 +41,8 @@ case class V2AppDefinition(
 
     uris: Seq[String] = AppDefinition.DefaultUris,
```

Comment:
Not 100% sure - but could we replace `uris` with a `def` and compute it from `fetch` (or even remove uris)?
The Format[V2AppDefinition] has to read/render uris and fetch.
Going this route, validation has to be done on the Json.reads function (see ipAddress as example).
With that, we basically replace uris as array of String with fetch as array of FetchUri's, which would make the code more clear and easy to understand.
What do you think?